tmt/steps/execute/internal.py

```diff
@@ -177,7 +186,12 @@ def execute(self, test, guest, progress, extra_environment):
             command = test.test
 
         # Prepare custom function to log output in verbose mode
-        def log(key, value=None, color=None, shift=1, level=1):
+        def log(
+                key: str,
+                value: Optional[Any] = None,
+                color: Optional[str] = None,
+                shift: Optional[int] = 1,
+                level: Optional[int] = 1) -> None:
```

Comment:
Parameter `value` should be `Optional[str]` as it is passed to [`verbose()`](https://github.com/teemtee/tmt/blob/main/tmt/utils.py#L453).

Params `shift` and `level` should both be `int`, they are never set to `None`.

Reply:
ty, changed.